README.md

```diff
@@ -112,4 +113,76 @@ tokenizer = AutoTokenizer.from_pretrained(model_id)
 pipe_cls = pipeline("text-classification", model=model, tokenizer=tokenizer)
 text = "He's a dreadful magician."
 outputs = pipe_cls(text)
-```
\ No newline at end of file
+```
+
+- Post-training quantizaiton example:
+```python
+from functools import partial
+from optimum.intel.openvino.quantization import OVQuantizer 
+from transformers import AutoTokenizer, AutoModelForSequenceClassification, OVModelForSequenceClassification
+
+model_id = "distilbert-base-uncased-finetuned-sst-2-english"
+model = AutoModelForSequenceClassification.from_pretrained(model_id)    
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+def preprocess_fn(examples, tokenizer):
+    return tokenizer(
+        examples["sentence"], padding="max_length", max_length=tokenizer.model_max_length, truncation=True
+    )
+
+quantizer = OVQuantizer.from_pretrained(model)
+calibration_dataset = quantizer.get_calibration_dataset(
+    "glue",
+    dataset_config_name="sst2",
+    preprocess_function=partial(preprocess_fn, tokenizer=tokenizer),
+    num_samples=100,
+    dataset_split="train",
+    preprocess_batch=True,
+)
+quantizer.quantize(
+    calibration_dataset=calibration_dataset,
+    save_directory='nncf_results' # use this folder as and input for OVModelForSequenceClassification
+                                  # to do the inference
+)
+
+optimized_model = OVModelForSequenceClassification.from_pretrained('nncf_results')
+```
+
+- Quantization-aware training example:
+
+```diff
+import numpy as np
+from datasets import load_dataset, load_metric
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, TrainingArguments, default_data_collator
+from optimum.intel.openvino.trainer import OVTrainer
++from optimum.intel.openvino import OVModelForSequenceClassification
+
+model_id = "distilbert-base-uncased-finetuned-sst-2-english"
+model = AutoModelForSequenceClassification.from_pretrained(model_id)    
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+dataset = load_dataset("glue", "sst2")
+dataset = dataset.map(
+    lambda examples: tokenizer(examples["sentence"], padding="max_length", max_length=128), batched=True
+)
+compute_metrics = lambda p: metric.compute(
```

Comment:
metric is not defined yet. Easiest is to add `metric = load_metric("accuracy")` but there is a DeprecationWarning that says to use evaluate instead. @echarlaix  should we add evaluate to nncf extra dependencies and use `metric = evaluate.load("accuracy")`?

Reply:
Yes it's a great point